### PR TITLE
feat: add a modal with information about what has changed in relation to the playground.

### PR DESCRIPTION
**Pull request by magicmatatjahu**
**Description**

Add a modal with information about what has changed in relation to the playground:
- modal opens when we have the query parameter `redirectedFrom=playground`

We need this to archive the playground repository as well as to make a redirect from playground to the studio - people should know what has changed in relation to the playground.

@derberg @alequetzalli Could you check the description of the changes?

Preview: https://deploy-preview-282--modest-rosalind-098b67.netlify.app/?redirectedFrom=playground

### CODE REVIEW

**Comment by mcturco** [comment]
Agreed that we should redirect users from the playground to the studio and archive that repo. 

I am wondering if this modal can be improved in the future to serve as an onboarding screen that delivers the new features in smaller digestible steps rather than listing them all out in one screen (things like this can seem overwhelming to users, I wonder how many will even read down the whole list?) 

Also just wondering what prompted the addition of this modal? Did someone show some confusion between playground and studio? I'm interested to see if this new modal is useful to users coming from the playground or if all of these new features are a bit more self-explanatory. 

Also curious as to what the usage will look like of this query parameter... will it only be added when redirected from the playground domain or is this something that we would maybe add to a button where first-time users of studio are more likely to click on it? Curious about how we can get this onboarding information across to more users other than just redirecting from playground without it becoming a nuisance for returning users. 

(Sorry if that was off-topic from the goals of this PR. Just kind of thinking out loud about how we could improve this modal and use it to the Studio's advantage in other ways than just one)

**Comment by magicmatatjahu** [comment]
@mcturco Thanks for that comment!

> I am wondering if this modal can be improved in the future to serve as an onboarding screen that delivers the new features in smaller digestible steps rather than listing them all out in one screen (things like this can seem overwhelming to users, I wonder how many will even read down the whole list?)

Yes, we can have the "changelog" like in (e.g.) Postman app to describe changes between versions of a studio. Could you create such issue? I don't know when we will implement it but it's worth keeping in mind.

> Also just wondering what prompted the addition of this modal? Did someone show some confusion between playground and studio? I'm interested to see if this new modal is useful to users coming from the playground or if all of these new features are a bit more self-explanatory.

Some people still use the playground and when we will enable redirection from playground to the studio, some people can be confused, so we need to list all changes in relation to the playground. So that list contains new features that we don't have in playground as well changes (that same features but working in different way),

> Also curious as to what the usage will look like of this query parameter... will it only be added when redirected from the playground domain or is this something that we would maybe add to a button where first-time users of studio are more likely to click on it? Curious about how we can get this onboarding information across to more users other than just redirecting from playground without it becoming a nuisance for returning users.

Only when people will be redirected. I wonder if we should save info that someone see that modal and don't show it again even if someone will redirected from playground again 🤔  For new users we can prepare such a tutorial what is something with highlighting relevant elements, but not in this PR.

**Comment by magicmatatjahu** [comment]
@derberg 

> It gives anxiety seeing so much text at first 😂

I know as a society we're going back to cuneiform writing (emoji), but I had a problem with it myself. It can be cured with a psychologist  :trollface:

**Comment by mcturco** [comment]
> Can we at least collapse this list of bullet points and offer a "Show what's changed" link? A simple one, something like a summary HTML tag, nothing fancy. It gives anxiety seeing so much text at first 😂

I have to agree with @fmvilas, I think this small change could improve the modal at least within this PR. But I will definitely create an issue for working on an onboarding process in Studio so we can work on that in the future 😄

**Comment by magicmatatjahu** [comment]
@mcturco @fmvilas I pushed changes:

image

WDYT?

**Review by mcturco — APPROVED** [review]
@magicmatatjahu Good idea! Definitely less overwhelming 👍  LGTM 🎉

**Comment by fmvilas** [comment]
Yeah, looks great to me. Just one more suggestion. Please change the button copy to "OK" instead of "Cancel" 🙏

File: src/components/Modals/RedirectedModal.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+    setShow(false);
+  }
+
+  function onShowMoreClick() {
+    setShowMore(true);
+  }
+
+  return (
+    <ConfirmModal
+      title='Welcome in the AsyncAPI Studio!'
```
Comment (fmvilas) [inline src/components/Modals/RedirectedModal.tsx:48]:
```suggestion
      title='Welcome to the AsyncAPI Studio!'
```

**Review by fmvilas — CHANGES_REQUESTED** [review]
And last thing. Sorry for the cluttered review 😄

**Review by fmvilas — APPROVED** [review]
🚀 Ship it! :)

**Comment by fmvilas** [comment]
@magicmatatjahu Just realized that after closing the modal, we leave the query param in the URL so if we refresh the page we get the modal again. It would be great to remove the query param (without refreshing) once we close the modal.

**Review by fmvilas — APPROVED** [review]

**Pull request merged**